presto-main/src/main/java/io/prestosql/server/security/JsonWebTokenAuthenticator.java

```diff
@@ -128,6 +132,17 @@ public Principal authenticate(HttpServletRequest request)
         }
     }
 
+    private String extractUser(String principal)
+            throws AuthenticationException
+    {
+        try {
+            return userExtraction.extractUser(principal);
+        }
+        catch (RuntimeException e) {
```

Comment:
`log.debug(e)`, in case the extraction fails for some "runtime" reason
(you know: regular exps...)

same in other places